browser/net/brave_reduce_language_network_delegate_helper.cc

```diff
@@ -69,6 +75,12 @@ int OnBeforeStartTransaction_ReduceLanguageWork(
                                              profile->GetPrefs())) {
     return net::OK;
   }
+  std::string tab_origin_etld1_host =
+      net::registry_controlled_domains::GetDomainAndRegistry(
+          ctx->tab_origin.host(),
+          net::registry_controlled_domains::EXCLUDE_PRIVATE_REGISTRIES);
+  if (kFarbleAcceptLanguageExceptions.contains(tab_origin_etld1_host))
```

Comment:
1) don't you want to check something like `DomainIs()`
2) can we add tests?

Reply:
Actually we want something like https://source.chromium.org/chromium/chromium/src/+/main:components/google/core/common/google_util.cc;l=126 but without the port checks. I've updated the code and added a test.